grunt_tasks/webpack.config.js

```diff
@@ -92,11 +130,13 @@ module.exports = {
             // JSON files
             {
                 test: /\.json$/,
+                include: loaderPaths,
```

Comment:
Nevermind, I think this probably is the easiest way. I thought it would be possible to just prepend loaders that a file matched, and that could be used to easily override core loaders, but [apparently that's not the case](http://stackoverflow.com/questions/32234329/what-is-the-loader-order-for-webpack), and that simply causes concatenation of matching loaders.